canonical/charmhub.io
# Rename /interfaces to /integrations

## Done
- rename `/interfaces/<interface-name>` to `/integrations/<interface-name>`
- redirect from `/interfaces/<interface-name>` to `/integrations/<interface-name>`
## How to QA
- Go to and click on any charm that has interfaces examples are: kafka, kafka-k8, click on the charm's integration tab, then click on any interface, ensure that it redirects to `/integrations/<interface-name>`
- Go to integrations/grafana_dashboard and ensure that it shows as expected, compare to [production](charmhub.io/interfaces/grafana_dashboard). Repeat for any other interface of your choice and ensure it works as expected.
- Go to interfaces/interface-name and ensure it redirects to `integrations/<interface-name>`
## Issue / Card
Fixes #

## Screenshots
